setup.py

```diff
@@ -1,16 +1,20 @@
 from setuptools import setup, find_packages
 
+
 def readme():
-    with open('README.rst') as f:
+    with open("README.rst") as f:
         return f.read()
 
-setup(name='protopipe',
-      version='0.1',
-      description='Pipeline to process events from DL0 to DL3',
-      url='http://github.com/jjlk/protopipe',
-      author='CEA',
-      author_email='julien.lefaucheur@cea.fr',
-      license='MIT',
-      packages=find_packages(),
-      install_requires=['ctapipe'],
-      zip_safe=False)
\ No newline at end of file
+
+setup(
+    name="protopipe",
+    version="0.2.1-dev",
+    description="Pipeline to process events from DL0 to DL3",
+    url="https://github.com/cta-observatory/protopipe",
+    author="CEA",
```

Comment:
cta-observatory?
Or authors read directly from GitHub.

Reply:
Yes, I forgot to update the authors: I can update the list from GitHub.

Reply:
Yes have a look into ctapipe